paasta_tools/tron_tools.py

```diff
@@ -148,14 +156,23 @@ def get_calculated_constraints(self):
             constraints.extend(self.get_pool_constraints())
             return constraints
 
+    def validate(self) -> List[str]:
+        # Use InstanceConfig to validate shared config keys like cpus and mem
+        error_msgs = super(TronActionConfig, self).validate()
```

Comment:
I don't totally understand this. If we are already inheriting from InstanceConfig, is the inherited method not enough?
Is the purpose here just to make the return errors prettier with the instance name?
If so I think it would be OK to put that in the InstanceConfig

Reply:
It used to call other methods that were on TronActionConfig (e.g. check_expected_runtime). I guess I left this here to leave room for that in the future.